pkg/sql/schemachanger/scexec/executor.go

```diff
@@ -14,22 +14,19 @@ import (
 	"context"
 
 	"github.com/cockroachdb/cockroach/pkg/sql/schemachanger/scop"
-	"github.com/cockroachdb/cockroach/pkg/util/log"
 	"github.com/cockroachdb/errors"
 )
 
 // ExecuteStage executes the provided ops. The ops must all be of the same type.
-func ExecuteStage(ctx context.Context, deps Dependencies, ops []scop.Op) error {
-	// It is perfectly valid to have empty stage after optimizations /
-	// transformations.
-	if len(ops) == 0 {
-		log.Infof(ctx, "skipping execution, no operations in this stage")
-		return nil
+func ExecuteStage(ctx context.Context, deps Dependencies, phase scop.Phase, ops []scop.Op) error {
+	typ := scop.MutationType
+	if len(ops) > 0 {
+		// It is perfectly valid to have an empty stage.
```

Comment:
Why? Shouldn't the planner trim such stages?

Reply:
Indeed. I think this is detritus from a previous attempt of mine at this point. I'll try and remove this.